stacks/apps/exchange.yml

```diff
@@ -1532,6 +1459,22 @@ Resources:
         - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
         - { Key: prx:dev:application, Value: Exchange }
       VisibilityTimeout: 5400 # 1.5 hour
+  ExchangePorterCallbackQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      DelaySeconds: 0
+      MessageRetentionPeriod: 345600 # 4 days
+      QueueName: !Sub ${Constants.SqsQueuePrefix}porter_callback
```

Comment:
The old queue is incorrectly named: `dc51b3fd_prod_exchange__exchange_porter_callback`
This creates a new correctly named: `dc51b3fd_prod_exchange_porter_callback`